pallets/gear/src/lib.rs

```diff
@@ -1756,7 +1769,7 @@ pub mod pallet {
         /// - `session_id`: id of the resume session.
         /// - `block_count`: the specified period of rent.
         #[pallet::call_index(11)]
-        #[pallet::weight(DbWeightOf::<T>::get().reads(1) + <T as Config>::WeightInfo::resume_session_commit(ProgramStorageOf::<T>::resume_session_page_count(session_id).unwrap_or(0)))]
+        #[pallet::weight(<T as Config>::WeightInfo::resume_session_commit())]
         pub fn resume_session_commit(
```

Comment:
Problem with weight: if some other session has already resumed program, then this will remove all pages data from storage. So, weight can depends on size of already created memory pages